src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -14,12 +14,15 @@ public sealed class PolicyContract : NativeContract
     {
         public override string Name => "Policy";
         public override int Id => -3;
+        public const uint DefaultBaseExecFee = 30;
 
+        private const uint MaxBaseExecFee = 10000;
```

Comment:
It's too large. Maybe `100` is enough.

Reply:
1000? I don't think it hurts us to have a bit bigger limit here.